media-source-respec.html

```diff
@@ -1160,15 +1223,67 @@ <h2><dfn>MediaSourceHandle</dfn> Object</h2>
         because {{MediaSource}} objects themselves are not transferable since they are event targets.</p>
 
+      <p>Each {{MediaSourceHandle}} object has a <dfn data-dfn-for="MediaSourceHandle">[[\has ever been assigned as
+        srcobject]]</dfn> internal slot that stores a {{boolean}}. It is initialized to false when the
+        {{MediaSourceHandle}} object is created, is set true in the extended {{HTMLMediaElement}}'s
+        {{HTMLMediaElement/srcObject}} setter as described in
+        <a href="#htmlmediaelement-extensions">HTMLMediaElement Extensions</a>, and if true, prevents successful
+        transfer of the {{MediaSourceHandle}} as described in the [=Transfer=] section.</p>
+
+      <p>{{MediaSourceHandle}} objects are {{Transferable}}, each having a <dfn
+        data-dfn-for="MediaSourceHandle">[[\Detached]]</dfn> internal slot that is used to ensure that once the handle
+        object has been transferred, it cannot be transferred again.</p>
+
       <pre class="idl">
-[Exposed=(Window,DedicatedWorker)]
+[Transferable, Exposed=(Window,DedicatedWorker)]
 interface MediaSourceHandle {};</pre>
 
-      <section><h2>Transfer</h2>
-        The {{MediaSourceHandle}} [=transfer steps=] and [=transfer-receiving steps=] require the implementation to
-        maintain an implicit internal slot referencing the underlying {{MediaSource}} to enable [=attaching to a media
-        element=] using {{HTMLMediaElement/srcObject}} and consequent setup of an attachment's [=cross-context
-        communication model=].
+      <section><h2><dfn>Transfer</dfn></h2>
+        <p>The {{MediaSourceHandle}} [=transfer steps=] and [=transfer-receiving steps=] require the implementation to
+          maintain an implicit internal slot referencing the underlying {{MediaSource}} to enable [=attaching to a media
+          element=] using {{HTMLMediaElement/srcObject}} and consequent setup of an attachment's [=cross-context
+          communication model=].</p>
+
+        <p class="note">
+          Implementors should be aware that assumption of "move" semantics implied by {{Transferable}} is not always
+          reality. For example, extensions or internal implementations of postMessage using broadcast may cause
+          unintended multiple recipients of a transferred {{MediaSourceHandle}}. For this reason, implementations are
+          guided to not resolve which potential clone of a transferred {{MediaSourceHandle}} is still valid for
+          attachment until and unless any handle for the underlying {{MediaSource}} object is used in the asynchronous
+          portion of the media element's resource selection algorithm. This is similar to the existing behavior for
+          attachment via [=MediaSource object URLs=], which can be cloned easily, where such a URL is valid for at most
+          one attachment start (across all of its potentially many clones).
+        </p>
```

Comment:
The above is just a note so it is fine for it to stay if you think it's useful, but it seems to dive into implementation-specifics that don't affect the externally visible behavior

Reply:
It is useful to me at least :)
Implementations may want to refer to this specific note internally when considering when to resolve potential clones.
We can remove later if more folks, especially implementors, believe this is a superfluous or implementation-specific note.